src/directions/types.ts

```diff
@@ -254,7 +254,8 @@ export interface MapLibreGlDirectionsConfiguration {
 
   /**
    * A minimal amount of pixels a waypoint or the hoverpoint must be dragged in order for the drag-event to be
-   * respected. Should be a number >= `0`. Any negative value is treated as `0`.
+   * respected, and for network requests to be made when using refreshOnMove = true. Should be a number >= `0`.
```

Comment:
Change please

> when using refreshOnMove = true.

to something like (check out similar places)

when {@link refreshOnMove|`refreshOnMove`} equals `true`.

Reply:
This is how TypeDoc works. Some magic and not a very clear comment, but turns out into a nicely-formatted documentation.